connect/mirror/src/test/java/org/apache/kafka/connect/mirror/MirrorConnectorsIntegrationTest.java

```diff
@@ -167,6 +182,35 @@ public void setup() throws IOException {
 
         primary.configureConnector("MirrorHeartbeatConnector", mm2Config.connectorBaseConfig(new SourceAndTarget("backup", "primary"),
             MirrorHeartbeatConnector.class));
+
+        waitUntilMirrorMakerIsRunning(primary, connectorNames);
+    }
+
+
+    private void waitUntilMirrorMakerIsRunning(EmbeddedConnectCluster connectCluster,
+        Set<String> connNames) throws InterruptedException {
+        for (String connector : connNames) {
+            TestUtils.waitForCondition(() -> areConnectorAndTasksRunning(connectCluster,
+                connector, MIN_TASKS), "Timed out trying to verify connector " +
+                connector + " was up!" );
+        }
+
+    }
+
+    private boolean areConnectorAndTasksRunning(EmbeddedConnectCluster connectCluster,
+        String connectorName, int numTasks) {
```

Comment:
nit: do we need the `numTasks` argument since the only caller is passing MIN_TASKS? Alternatively, we could replace `info.tasks().size() >= numTasks` with `!info.tasks().isEmpty()` and get rid of `MIN_TASKS`.

Reply:
That's a good idea, I'll replace it with `!info.tasks().isEmpty()` and get rid of `MIN_TASKS`.